resources/fixtures/JB-Systems-The-WinnerII.qxf

```diff
@@ -62,10 +62,10 @@
  </Channel>
  <Mode Name="Mode 1">
   <Physical>
-   <Bulb Lumens="0" Type="ELC 250W" ColourTemperature="0"/>
-   <Dimensions Width="200" Height="155" Weight="10" Depth="500"/>
+   <Bulb Type="ELC 250W" Lumens="0" ColourTemperature="0"/>
```

Comment:
According to some online calculator, 435 lux @ circle with 1,15 radius (4,15 sq m) is 1800 lumens